examples/java8/src/test/java/org/apache/beam/examples/snippets/SnippetsTest.java

```diff
@@ -42,7 +45,29 @@
 public class SnippetsTest implements Serializable {
 
   @Rule
-  public transient TestPipeline p = TestPipeline.create();
+  public final transient TestPipeline p = TestPipeline.create();
+
+  @Test
+  public void testModelBigQueryIO() {
+    // We cannot test BigQueryIO functionality in unit tests, therefore we limit ourselves
+    // to making sure the pipeline containing BigQuery sources and sinks can be built.
+    boolean runLocally = false;
```

Comment:
Are lines 56-65 dead code?

Reply:
Basically yes, but they are useful when testing locally. Replacing the `project`, `dataset` and `table` variables with whatever you're using allows for testing locally and making sure everything is doing what it's supposed to do